content/src/main/content/jcr_root/etc/tags/_rep_policy.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is there a way to deploy this permission in a way that only works if the /etc/tags folder already exists?  When I deploy this to AEM 6.4, the next time I restart the AEM server it sees the /etc/tags folder and assumes my tags are there instead of under /content/cq:tags.

Reply:
Maybe I just need to switch over to using `EnsureServiceUser`...

Reply:
This is now resolved.